GOVERNANCE.md

```diff
@@ -57,39 +60,60 @@ decision on a matter.
 
 While this approach has served us well, as the Project grows and faces more
 legal and financial decisions and interacts with other institutions, we see a
-need for a more formal governance model. Moving forward The Project leadership
-will consist of a BDFL and Steering Council. We view this governance model as
-the formalization of what we are already doing, rather than a change in
-direction.
+need for a more formal governance and organization model.
+We view this governance model as the formalization of what we are already doing,
+rather than a change in direction.
+
+## Community and Team Architecture
+The PyMC community is organized in an onion-like fashion.
+The tiers relevant to the project governance are listed below sorted by
+increasing responsibility. Due to the onion-like structure, members of a group are
+also members of all the groups listed above:
+
+* Contributors
+* Recurring Contributors
+* Core Contributors
+* Steering Council
+* BDFL
+
+Recurring Contributors comprise what we understand as the PyMC Team.
+The Team will generally act as a single unit, except for some specific
+questions where dedicated teams will prevail.
+The PyMC project currently counts with the Developer and Documentation teams.
+Team members can be part of one, some or none of these dedicated teams.
+
+![community_diagram](docs/community_diagram.png)
```

Comment:
Question: can you be a core contributor without being part of either the dev team or the doc team? If not, the diagram would need some tweaking.

Reply:
Yes, one can be core contributor or steering council member without being on any of the teams